controllers/protectedvolumereplicationgrouplist_controller.go

```diff
@@ -50,31 +50,33 @@ type ProtectedVolumeReplicationGroupListInstance struct {
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.9.2/pkg/reconcile
 func (r *ProtectedVolumeReplicationGroupListReconciler) Reconcile(ctx context.Context, req ctrl.Request,
 ) (ctrl.Result, error) {
-	log := ctrl.Log.WithName("controllers").WithName("protectedvolumereplicationgrouplist").
-		WithValues("name", req.NamespacedName.Name)
-
-	log.Info("protectedvolumereplicationgrouplist reconcile start")
-
 	// save all the commonly used parameters in a struct
 	s := ProtectedVolumeReplicationGroupListInstance{
 		reconciler: r,
 		ctx:        ctx,
-		log:        log,
+		log:        ctrl.Log.WithName("pvrgl").WithValues("name", req.NamespacedName.Name),
 		instance:   &ramendrv1alpha1.ProtectedVolumeReplicationGroupList{},
 	}
 
+	s.log.Info("reconcile start")
+	defer s.log.Info("reconcile end")
```

Comment:
For future change - starting/entering and ending/exiting is an important concept in controllers, so we should have special log notation to make it easy to navigate the log.

Also the time spent in a reconcile is important, we want to keep reconcile fast and know about slow ones.

This is common to many servers. In oVirt we had this format:

    START {request info} ...
    FINISH {request info} in xxx seconds

We do the same here:

    ENTER {reconcile context}
    EXIT in xxxx seconds {reconcile context}